networkx/generators/mycielski.py

```diff
@@ -0,0 +1,114 @@
+#    Copyright (C) 2010-2013 by
+#    Aric Hagberg <hagberg@lanl.gov>
+#    Dan Schult <dschult@colgate.edu>
+#    Pieter Swart <swart@lanl.gov>
+#    All rights reserved.
+#    BSD license.
+
+"""Functions related to the Mycielski Operation and the Mycielskian family
+of graphs.
+
+"""
+
+import networkx as nx
+from networkx.utils import not_implemented_for
+
+__all__ = ['mycielskian', 'mycielski_graph']
+
+
+@not_implemented_for('directed')
+@not_implemented_for('multigraph')
+def mycielskian(G, iterations=1):
+    r"""Returns the Mycielskian of a simple, undirected graph G
+
+    The Mycielskian of graph preserves a graph's triangle free
+    property while increasing the chromatic number by 1.
+
+    The Mycielski Operation on a graph, `G=(V,E)`, constructs a new graph
```

Comment:
Backticks should be used for references to function arguments. For math mode, use the `math` role, like this: ``` :math:`G = (V, E)` ```. However, don't use math mode if italics will do.